Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM registry.redhat.io/ubi8/s2i-base
```

Comment:
i noticed that this makes travis fail since it was asking for a login.  i wonder if there is a "public" version somewhere

Reply:
Yeah, I'm going to try to find something that does not require a login. I thought it would be available to anyone so there might be a different repository available to pull from.

Reply:
Perhaps this is intentional in that we can publish the resulting image (not done for rpms, but we do that for the s2i container) to dockerhub, but the ubi8 base image itself is available in the `registry.access.redhat.com` which requires a login. In this case we need to configure travis with for this. @lance @balunasj Does the above sound right to you or am I missing something?

@lholmquist could you try this locally to verify that the tests pass for you and in the mean time

Reply:
Forget that, I'm an idiot. I should have been using `registry.access.redhat.com` instead :(